### ISSUE TITLE
Fix HTML on Learn page

### ISSUE THREAD

**Issue by MelSumner** [emberjs/website#3155]
**Context:** 
Some of the bullet-ed items on the learn page are not contained within an `<ul>` element. As a result, the page is failing an accessibility conformance test. 

**To fix:** 
Add a `<ul>` around the `<li>` elements on the learn page (source/learn.html.erb)

**Screenshot:** 
![image](https://user-images.githubusercontent.com/4587451/34950372-9c0a1c36-f9d8-11e7-9dd2-2507381f6bdd.png)

**Comment by dshiling** [comment]
Hey! I can take this one if still open.

**Comment by locks** [comment]
It is, go ahead! :)
